components/operator/README.md

```diff
@@ -170,7 +170,9 @@ Available settings:
 
 | Key                                                                                      | Type   | Example             | Description                                                          |
 |------------------------------------------------------------------------------------------|--------|---------------------|----------------------------------------------------------------------|
-| aws.service-account                                                                      | string |                     |  AWS Role |
+| aws.service-account                                                                      | string |                     |  AWS Role                                                            |
+| licence.token                                                                            | string |                     | Token to check licence                                               |
```

Comment:
These parameters are still settings?

Reply:
No you're right, I'm gonna remove them thanks